promotion/src/main/java/com/yas/promotion/exception/ApiExceptionHandler.java

```diff
@@ -81,6 +81,13 @@ protected ResponseEntity<ErrorVm> handleDuplicated(DuplicatedException e) {
         return ResponseEntity.badRequest().body(errorVm);
     }
 
+    @ExceptionHandler(InvalidDateRangeException.class)
+    protected ResponseEntity<ErrorVm> handleInvalidDateRange(InvalidDateRangeException e) {
+        ErrorVm errorVm = new ErrorVm(HttpStatus.BAD_REQUEST.toString(),
+            HttpStatus.BAD_REQUEST.getReasonPhrase(), e.getMessage());
+        return ResponseEntity.badRequest().body(errorVm);
+    }
+
```

Comment:
There is already a handler method for bad request errors. I think you can reuse that by making `InvalidDateRangeException` inherit from `BadRequestException`. This way, you won't need to create a duplicate handler like this.